### PR TITLE
Reverts Pico manipluator costing materials

### PR DESCRIPTION
**Pull request by jakmak3**
I had originally made this to compliment a buff to fabricators but as it turns out community was against this so i am undoing it. The pico manip just costs metal now. Also i reduced the price of the RPED since it would take well up to 2 minutes to build one with the old price to where it now takes only 40 seconds to build.

### CODE REVIEW

**Review by PJB3005 — APPROVED** [review]

**Comment by Kurfursten** [comment]
Stage one PR: remove the mineral cost from the RPED, but in exchange make it cost more metal and glass.
Stage two PR: reduce the metal and glass, but do not re-add roid minerals

Stage one PR: change the cost reduction stock part to the manipulator, but in exchange make it cost minerals
Stage two PR: remove the mineral cost from manipulators but do not change the manipulators back to only affecting speed

We need a powercreep tag

**Comment by jakmak3** [comment]
OK kurf tell me exactly what the hell i should do regarding pico manips. I only added in the mineral price to it because i thought that if i didnt then the original fabricator and lathe part buff would not have been merged. The community clearly hates it so if i remove the mineral price the community would enjoy it but would result in powercreep. If i keep it then powercreep stops but people get fed up with it.

**Comment by Kurfursten** [comment]
Maybe we should get someone who knows why people are apparently(?) so angry then. Is it because they don't know you can finish R&D with matter bins? Is there just some super powerful Xenobiologist bloc that wants pico manipulators for perfect monkey recycling at shift start? (And if so, I want my backpay in redditvotes for adding that upgrade)

**Comment by Probe1** [comment]
Yea no you don't get to revert just the nerfs.  Either it all goes or it stays.

**Comment by Probe1** [comment]
After reviewing it in game the change in the original PR isn't so hefty that it needs extensive DNM review.

**Comment by yclatious** [comment]
If the whole thing is removed or not, I couldn't give a hoot, but making picos take minerals is just pants on head retarded and adds nothing to the game, its needless restriction for the sake of a convenient item being more convenient to make, alongside fucking up the monkeys for xenobio and making picos no longer be used by mechanics, and almost nobody plays mechanics already, screwing them of the best and most accessible parts is dumb. Even if just for the xenobio folks, Id say to merge it, whether it means to remove the RPD mineral changes or not, people can just decon the machines and do it manualy, no need for a fancy toy.

**Comment by Kurfursten** [comment]
If we're balancing all stock parts around the monkey recycler, I feel like I deserve to be some sort of xenobiology celebrity for adding that upgrade, but the selection of the manipulators was arbitrary. We could just as easily change it to another stock part used for monkey efficiency (especially now that the uses for manipulators and lasers were reversed in other machines, it would make sense there too).

The tiers are there for a reason - tier 1, autolathe and vending, anyone can make it. Tier 2 needs a protolathe, but just metal. Tier 3 requires mining. Breaking away from that consistency is, to quote you, "dumb" and "pants on head retarded". Or can you explain a situation in which you ever make nano manipulators? Did you even know they exist?

untitled

**Comment by yclatious** [comment]
Hey, its a ultra useful machine for a niche but enjoyable job that this change made worse, you want credit for the pico thing, congrats, you did did good, party confetti in your post.

If you want to make it a new piece to do the effect, its a wyci, but I don't see a issue that we ever had with t3 manipulators being a thing, haven't heard a single person complaining about having them be iron only or any problem surging from it despite breaking the theme you mentioned. Dumb it may be, but if execution is nigh flawless, its dumb but it works. And if it works, you either leave it alone or you improve it, can't improve it other than making the higher tier pieces do better effects in machines, and when that's a thing, youl make them miner only, which makes people pet peved when its going to change things like mechanics being mostly useless till miners get them minerals and xenobio not getting unlimited monkeys unless t2 makes that, and Im sure that itself counts as powercreep, right?

I knew nano were a thing, but I never used them, because manipulator changes now and then are still mostly irrelevant, and why get t2 when you can get t3, not like t3 is any real good or will influence anything major, but from 0.2 and 0.3 for slight metal costs, I know what Ill pick. Certain tiers will always be neglected, either because mechanic will one day be actually good and his upgrades will matter (sooner would byond die or a remake would happen),  or people just wait till the newest tier is available instead of wasting time with less effective and sometimes with much shittier results. Even in infinitely better designed games with actual developers and such being on a payroll to make sure design flaws don't happen and progression is smooth and predictable, tiers can be skipped. In SS13, with rounds that can range from a simple hour and up to 3 before admins go nuts with the fun buttons, you have to realize that playing a role based around improving the station, you will naturally skip tiers of machine parts when you know all your effort will be nulled in the next 3 hours and the previous parts are simply not worth using, there is no permanence, so expecting players to strictly adhere to tier lists and follow them as they should will simply not happen, especially when they impact the station and the machines so little, the job is barely even worth playing, minus its saving bandaid fix, the flatpack fabricator.

tldr, shit was fine as it was with iron only in t3, its dumbfuck retarded to break thematic progression and materials but shit works anyways, so let it. Don't expect to be able to balance things perfectly in ss13 either, its a hopeless dream since its so different from most other games, and even they fuck up balancing and progression. Make it iron again and make the rpd into rarely gotten shit, fixes xenobio without making you remake parts, good for everyone.

**Comment by PJB3005** [comment]
Personally I think material progression should go like this:

1. Tier 1 is cheap as shit and available round start.
2. Tier 2 is a bumps up the cost 3x, not a big deal but whatever. Need 15 minutes in R&D for these.
3. Tier 3 bumps up the price to significant levels, needing minerals. Probably like 30 minutes in R&D depending on how the crew helps out.
4. Tier 4 is "o shit" level, need significant materials (talking about a diamond for like ~5 parts), but are extremely useful. 1 hour in R&D.

Of course, such balancing would be accompanied by an R&D rework which...

**Comment by yclatious** [comment]
will never happen, achieve levels of tedium bigger than what we currently have, no doubt make it needlessly more complex and take forever to make by people and then make mechanics even more useless than now, with them then being hailed as nothing but glorified flatpack producers, even more than now.

¯\_(ツ)_/¯

**Comment by SonixApache** [comment]
why would t2 parts even exist if you can get a much better upgrade for slightly more of the same materials

mechanics are a meme

**Comment by PJB3005** [comment]
@yclatious in some cases tiers are meant to be skipped, sure. That doesn't mean that in every case tiers should be skipped though.

You're basically admitting you're too short sighted to even try to think of a solution, and would rather leave everything alone because it's "fine" right now.

**Comment by yclatious** [comment]
Didn't say they always should, Im saying making it completely balanced is a pipedream, and accepting this particular case won't do anyone any harm and only bring benefits, as I don't remember a single issue that metal only picos ever brought, only slight advantages, in some cases, a lot of commodity, like the monkey grinder. Breaks the theme, but works, let it slide, essentially.

By my memory, the general idea of a rework you talked about was essentially all about adding extra steps to it, right? Things like making some prototypes or sumwhat, making research timegated in some manner too? By my memory, it was a generally well received basis for it, which I can already tell would make research from a boring 5 minutes linear chore that allows potential for greatness into a more mechanically varied but still boring 20 minutes of work, a slightly less linear chore that will then allow slightly better tools that realistically, science will never use and by then, mechanics would have been petitioned to be removed due to no worthwhile part buffs.

Im not really a pessimist, but from what you have established to be your core idea of a RnD revamp, I can't see it ending up much better, only worse in boredom of actually doing rnd, whilst admittedly having a lot more potent toys and buffs to it. I admit, I can't think of a better structure to it yet or have I really tried, but ill stand by what I say, till you can find a better way of doing things, don't change, otherwise you are revamping a system and making it worse for the sake of change, not for the sake of improving it and using change to do it.

**Comment by aceedex** [comment]
But what exactly is the issue here? Outside of machine upgrading the manip. Is useless except for materials 6, which you can do with the super matter bin

**Comment by jakmak** [comment]
What if we had a T4 that costs diamond and a T5 that costs phazite. Of course T5 should be borderline gamebreakingly OP (10 second cloner, shit out 3 Durand's in 25 seconds, entire armory made by a protholathe with 2 sheets of Matt's) and would be balanced out by the phazite price

**Comment by PJB3005** [comment]
There is no way to balance mineral costs unless you make a mineral not able to work in borg mat synths.

**Comment by Probe1** [comment]
1.  why do we want game breaking upgrades @jakmak 

2.  material synths were a mistake @PJB3005

**Comment by jakmak3** [comment]
As in probe t4 being the common end goal (diamond) and t5 being ultra autism. I didnt mean litterally game breaking but instead really really powerfull and something to strive for to achieve.

**Comment by Kurfursten** [comment]
Since diamond is brought in at the same time as everything else (for muh drill), it only really works if you charge a ton, like a whole sheet of diamond for one part. Otherwise we're back to the problem of "you've made the previous tier irrelevant"

**Comment by TheFriggenSawbones** [comment]
🍆

**Comment by ComicIronic** [comment]
At one time I considered making it possible to upgrade the flatpacker to automatically upgrade machines it produced, but the issue is that it would either require material costs on flatpack designs, or make a very strong flatpacker.

**Comment by Megamaw** [comment]
i feel like the progression system of 'oh rare mining minerals' doesnt really work

because at no point in the round do you start getting more diamonds. miners don't suddenly start getting more diamonds later in the round.

you're not really earning it, because there's really very little a scientist or mechanic can do to get more diamonds, excepting special circumstances.

tl;dr make it hard, but something scientists can actually do something to work towards instead of hoping to god there's some miners that actually give a shit that round

**Comment by PJB3005** [comment]
> miners don't suddenly start getting more diamonds later in the round.

Yes they do. Diamond drill makes you fast so you can find minerals more easily.

**Comment by Megamaw** [comment]
@PJB3005 do you actually play the game because if you can get miners to do more than one or two escursions you must be a miracle worker

**Comment by PJB3005** [comment]
@Megamaw not something we should balance around here.

**Comment by Probe1** [comment]
Lets revisit this in a month and see if players are livid over the changes or have accepted it.

**Comment by Probe1** [comment]
After the initial shock of the change everyone stopped caring.  Who saw that coming?
